lib/Doctrine/DBAL/Schema/TableDiff.php

```diff
@@ -130,25 +129,16 @@ public function __construct(
 
     /**
      * @param AbstractPlatform $platform The platform to use for retrieving this table diff's name.
-     *
-     * @return Identifier
      */
-    public function getName(AbstractPlatform $platform)
+    public function getName(AbstractPlatform $platform) : Identifier
     {
         return new Identifier(
             $this->fromTable instanceof Table ? $this->fromTable->getQuotedName($platform) : $this->name
         );
     }
 
-    /**
-     * @return Identifier|false
-     */
-    public function getNewName()
+    public function getNewName() : ?Identifier
     {
-        if ($this->newName === false) {
-            return false;
-        }
-
-        return new Identifier($this->newName);
+        return $this->newName !== null ? new Identifier($this->newName) : null;
```

Comment:
I'd rather leave the `if` instead of the ternary for better readability and more accurate code coverage.